projects/inih/parse_fuzzer.dict

```diff
@@ -0,0 +1,4 @@
+"["
```

Comment:
it would be the best if you could provide a seed corpus: https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#seed-corpus

Reply:
* Which seed corpus size is recommended?
* Should this corpus be real valid ini files or it can be copied from LibFuzzer local run?

Reply:
> Which seed corpus size is recommended?

The more is better, but please minimize it first using `-merge=1` option, e.g.:

```bash
mkdir seed_corpus_minimized
./fuzzer -merge=1 -use_value_profile=1 ./seed_corpus_minimized <path_to_directory_with_original_seed_corpus>
```

> Should this corpus be real valid ini files or it can be copied from LibFuzzer local run?

It should be real valid ini files, please don't add files generated locally by libFuzzer as OSS-Fuzz we'll generate a tons of them anyway.

Reply:
Can I store an archive with seed corpus in oss-fuzz/projects/inih directory?
Should I add additional lines to build.sh?

Reply:
3.5KB is fine to store here.

`build.sh` should copy the corpus archive into `$OUT` with a name like `<fuzz_target_name>_seed_corpus.zip`, see https://github.com/google/oss-fuzz/blob/bb78fa98278babaf3fd2c509b0c956d976bf8d4d/docs/new_project_guide.md#seed-corpus

Reply:
Done